### PR TITLE
colours and unique naming

### PR DESCRIPTION
**Pull request by DBJDBJ**

### CODE REVIEW

File: ubench.h [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /*
    The latest version of this library is available on GitHub;
-   https://github.com/sheredom/ubench.h
+   https://github.com/sheUBENCH_REDom/ubench.h
```
Comment (sheredom) [inline ubench.h:3]: This looks like a search and replace gone wrong!

Comment (DBJDBJ) [reply ubench.h:3]: Ah ok let me look into it now ...

**Comment by DBJDBJ** [comment]
testing ubench and utest together so had to make colour enums unique
```cpp
enum ubench_colours_enum
{
  UBENCH_RESET,
  UBENCH_GREEN,
  UBENCH_RED
};
```

**Comment by DBJDBJ** [comment]
Ok I looked into CI failures, I understand ... what a juvenile mistake ... let me fix it

**Comment by sheredom** [comment]
@DBJDBJ ock don't worry about it - I make these mistakes a hundred times a day!

**Pull request closed without merging**